R/fda-table_8.R

```diff
@@ -0,0 +1,56 @@
+# Pre-processing
+library(dplyr)
+library(scda)
+library(tern)
+library(rlistings)
+
+# Deaths
+adae <-
+  synthetic_cdisc_dataset("rcd_2022_06_27", "adae") %>%
+  filter(SAFFL == "Y" & AESDTH == "Y") %>%
+  select(
+    STUDYID, USUBJID, ACTARM, ACTARMCD, AGE, SEX, AETERM, AEDECOD, ASTDY, AESDTH, AEOUT, DTHADY
+  ) %>%
+  mutate(AGESEX = paste0(AGE, "/", SEX)) %>%
+  distinct(USUBJID, AETERM, AEDECOD, DTHADY, .keep_all = TRUE)
```

Comment:
Same variable replacement as above
```suggestion
  distinct(USUBJID, DTHCAT, DTHCAUS, DTHADY, .keep_all = TRUE)
```